README.md

```diff
@@ -10,7 +10,22 @@ First, you'll need to configure a database. Once this project is deployed, we'll
 
 ### Windows setup
 
-Coming Soon!
+## Chocolatey and Postgres (Needs Testing)
+1. [Install Choclatey](https://chocolatey.org/docs/installation#installing-chocolatey) if you do not already have it installed.
+2. Add `C:\ProgramData\chocolatey\bin` to your paths (type path in search, click Edit environment varaibles for you account).
+3. In your cmd.exe (admin) terminal run `choco install postgresql --params '/Password:<assigned to postgres user>` (cmd.exe (admin) can be found by pressing `win + x`).
```

Comment:
On my windows 10 vm using admin powershell (I know that wasn't the instruction, but whatever), I ran `choco install postgresql --params '/Password:getmein'` and it still prompted me for a password for the postgres user. Not sure what happened there

Reply:
The hiccups I ran into were with the password.  Maybe it would be better to just let the terminal prompt the user. Thoughts?

Reply:
Also, it should be powershell because that is what is listed in the win + x menu.